source/components/edit.html

```diff
@@ -49,5 +50,17 @@ <h2>Notes</h2>
         <textarea id="edit-notes" placeholder="Enter notes here"></textarea>
       </div>
     </div>
+    <!-- edit warning, must fill in the name-->
+    <div id="edit-button-box" class="edit-box">
+      <form class = "edit-box-content">
```

Comment:
can we turn `class = "edit-box-content"` to `class="edit-box-content"` (no spaces around the = )